components/mpas-ocean/cime_config/buildnml

```diff
@@ -1110,6 +1110,8 @@ def buildnml(case, caseroot, compname):
         lines.append('    <var name="vertVelocityTop"/>')
         lines.append('    <var name="zMid"/>')
         lines.append('    <var name="atmosphericPressure"/>')
+		lines.append('    <var name="normalMLEvelocity"/>')
+		lines.append('    <var name="vertMLEVelocityTop"/>')
```

Comment:
```suggestion
        lines.append('    <var name="normalMLEvelocity"/>')
        lines.append('    <var name="vertMLEVelocityTop"/>')
```

Reply:
Thanks for adding theses!  They are needed for the offline MOC and are likely to be useful for debugging.

Reply:
We need to add them to standalone as well.  We want to maintain consistency:
https://github.com/E3SM-Project/E3SM/blob/master/components/mpas-ocean/src/analysis_members/Registry_time_series_stats_monthly_mean.xml#L135-L136

It seems like we probably want to enable `config_submesoscale_enable = .true.` in compass tests, too, following this PR.

Reply:
Done!